src/concat.js

```diff
@@ -30,8 +30,11 @@ var toString = require('./toString');
  *      R.concat('ABC', 'DEF'); // 'ABCDEF'
  */
 module.exports = _curry2(function concat(a, b) {
+  if (a == null || !is(Function, a.concat)) {
```

Comment:
Let's use `_isFunction` rather than `is(Function)` to avoid dependence on `instanceof` (#1756).